### PR TITLE
feat(messages): chat list

### LINKED ISSUE #204: feat(messages): messages list
# Description
As a user, we want to see the messages a specific chat room and be able to create new messages.

# Acceptance criteria
- [ ] Message component created
- [ ] List of messages components created
- [ ] Group messages by day
- [ ] Pagination with infinite scroll
- [ ] Fetch data from the backend

# Design reference
Screen Shot 2022-05-31 at 20 19 39

### LINKED ISSUE #221: feat(messages): create component to list chats
# Description
Create component to list the chats

# Acceptance criteria
- Should be able to select a chat room

# Design reference
Screen Shot 2022-06-04 at 19 52 24

### PR DESCRIPTION
**Pull request by JoseNoriegaa**
# Description
This PR adds a component to list the chat rooms

# Changes
- Resolves #221 

# Preview
Screen Shot 2022-06-02 at 17 06 55

### CODE REVIEW

File: packages/messaging-system/src/components/ChatList/ChatList.module.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.listItemContainer {
+  width: 100%;
+  list-style: none;
+  margin-bottom: 24px;
+}
+
+.listItem {
+  background: none;
+  color: inherit;
+	border: none;
```
Comment (JoseNoriegaa) [inline packages/messaging-system/src/components/ChatList/ChatList.module.scss:28]: I don't know what happen here, but I'll fix it on my next PR

**Review by hecttoressau — APPROVED** [review]
LGTM

File: packages/messaging-system/src/components/ChatList/ChatList.module.scss (hunk 1 above)
Comment (arselt) [reply packages/messaging-system/src/components/ChatList/ChatList.module.scss:28]: What is wrong here?

**Review by arselt — APPROVED** [review]
LGTM2

File: packages/messaging-system/src/components/ChatList/ChatList.module.scss (hunk 1 above)
Comment (JoseNoriegaa) [reply packages/messaging-system/src/components/ChatList/ChatList.module.scss:28]: indentation

Comment (arselt) [reply packages/messaging-system/src/components/ChatList/ChatList.module.scss:28]: So weird, in vscode everything seems fine.
But yes, it is a just weird ident.

**Pull request merged**